project/apps/matching-service/src/matching.gateway.ts

```diff
@@ -37,6 +38,7 @@ export class MatchingGateway
   constructor(
```

Comment:
unrelated to current changes: Would it be good to just add in something like
```
  afterInit(server: Server) {
    this.logger.log('WebSocket gateway initialized');
  }
```
for the purpose of sanity check?